dfm/data/load.py

```diff
@@ -187,3 +204,37 @@ def load_dfm_dataset(dataset: str, **kwargs) -> Dataset:
         raise ValueError(
             f"invalid dataset. Valid datasets include {dataset_loaders.keys()}"
         )
+
+
+def load_multiple_dfm_datasets(datasets: List[str], **kwargs) -> Dataset:
+    datasets = [load_dfm_dataset(dataset, **kwargs) for dataset in datasets]
+    return datasets.concatenate_datasets(datasets)
+
+
+#### TEMPORARY TO TEST TRAIN SCRIPT
```

Comment:
Good for now, but we should take a closer look at dataset loading.

Reply:
Agreed.